client/src/pages/LoadingPage.test.jsx

```diff
@@ -1,52 +1,105 @@
 import React from "react";
-import { render, waitFor } from "@testing-library/react";
-import { afterAll, beforeEach, describe, test, vi, expect } from "vitest";
+import { render, waitFor, screen } from "@testing-library/react";
+import { afterEach, describe, test, vi, expect } from "vitest";
 import { ThemeProvider } from "@mui/material";
 import MinutesContext from "../contexts/MinutesContext";
-import { mockMinutesContextState } from "../util/test.helpers";
+import {
+  mockGetMinutesResponse,
+  mockMinutesContextState,
+} from "../util/test.helpers";
 import theme from "../theme";
 import LoadingPage from "./LoadingPage";
+import logoImage from "../assets/images/logo.png";
+import minutesService from "../services/minutesService";
 
 const mockNavigate = vi.fn();
+const mockToken = "mocktoken";
+const updateMetadataMock = vi.fn();
+const updateMinutesaMock = vi.fn();
```

Comment:
small typo here, should be updateMinutesMock not updateMinutesaMock

Reply:
Good catch!